### PR TITLE
Use common modal popup utility

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Bumped origin-web-common to version 0.0.44
Bumped patternfly to version 3.26.1
Bumped angular-patternfly to version 4.3.0

### CODE REVIEW

File: src/styles/projects-summary.less [hunk 1]
```diff
@@ -146,91 +158,18 @@
...
-      display: block;
-      border-color: transparent transparent @color-pf-black-150;
-      border-style: solid;
-      border-width: 0 7px 7px;
-      height: 0;
-      left: (@grid-gutter-width / 4);
-      position: absolute;
-      top: -7px;
-      width: 0;
+.catalog-edit-project {
```
Comment (rhamilto) [inline src/styles/projects-summary.less]: Nit:  alpha.  This rule should come first in the file.

File: src/styles/projects-summary.less [hunk 2]
```diff
@@ -146,91 +158,18 @@
...
+  .origin-modal-popup {
```
Comment (rhamilto) [inline src/styles/projects-summary.less]: Nit:  this doesn't need to be nested.

**Comment by jeff-phillips-18** [comment]
Fixed the nits. Thanks @rhamilto

**Comment by jeff-phillips-18** [comment]
Minor changes for on-close callback type and shown variable names so they aren't confused with a function name.

**Comment by jeff-phillips-18** [comment]
@spadgett removed WIP status, bumped versions of common, patternfly, angular-patternfly

**Comment by spadgett** [comment]
Fixes #1900, correct?

**Comment by spadgett** [comment]
Seeing Travis failures.

```
Firefox 31.0.0 (Linux 0.0.0) Projects Summary Panel should show the create project panel when Create Project button is clicked FAILED
	Expected 0 to be 1.
	@/home/travis/build/openshift/origin-web-catalog/test/test-bundle.ts:9:1
.
Firefox 31.0.0 (Linux 0.0.0) Projects Summary Panel should show the edit project panel when the Edit menu item is clicked FAILED
	Expected 0 to be 1.
	@/home/travis/build/openshift/origin-web-catalog/test/test-bundle.ts:9:1
```

**Comment by jeff-phillips-18** [comment]
@spadgett yes, the angular patternfly upgrade will fix #1900 when we include it in console.

**Comment by jeff-phillips-18** [comment]
@spadgett Fixed unit tests.

**Pull request merged**